mensor/measures/types.py

```diff
@@ -777,20 +776,25 @@ def matches(self, unit_type, reverse=False):
 class _Measure(_ProvidedFeature):
 
     def __init__(self, name, expr=None, default=None, desc=None,
-                 distribution='normal', shared=False, provider=None):
-        _ProvidedFeature.__init__(self, name, expr=expr, default=default, desc=desc, shared=shared, provider=provider)
+                 distribution='normal', shared=False, provider=None, **attrs):
+        _ProvidedFeature.__init__(
+            self, name, expr=expr, default=default, desc=desc, shared=shared, provider=provider,
+            **attrs
+        )
         self.distribution = distribution
 
     def transforms_for_unit_type(self, unit_type, stats_registry=None):
-        transforms = {
+        transforms = {  # defaults
             'pre_agg': None,
             'agg': 'sum',
             'post_agg': None,
             'pre_rebase_agg': None,
             'rebase_agg': 'sum',
             'post_rebase_agg': None
         }
+
         if isinstance(self.transforms, dict):
+            transforms.update(self.transforms.get('_default', {}))
```

Comment:
default transforms for all unit types?

Reply:
I remain unconvinced, I'm afraid, that this is a good idea, as it leads to confusing (and usually wrong) behaviour :/. Suppose we chose `count` as the default aggregation for some measure. That might make sense the first time it is aggregated, but on subsequent aggregations (which may occur due to multiple rebase operations, for example), you would be taking the count of counts at each aggregations, which would make the resulting measure likely meaningless.